<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadEditionPageWizard</class>
 <widget class="QWizardPage" name="UploadEditionPageWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_datastore">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Ubuntu Condensed&quot;;
color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>Espace de travail</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="DatastoreComboBox" name="cbx_datastore"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_files_put">
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Ubuntu Condensed&quot;;
color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>Récuperez votre fichier de données</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QgsFileWidget" name="flw_files_put">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbl_files_summary">
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Ubuntu Condensed&quot;;
color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>Fichiers à importer :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QListWidget" name="lvw_import_data">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::ActionMask</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbl_data">
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Ubuntu Condensed&quot;;
color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>Nommez votre donnée :  </string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="lne_data">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly|Qt::ImhUrlCharactersOnly</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lne_projection">
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Ubuntu Condensed&quot;;
color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>Indiquez la projection de vos données :</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QgsProjectionSelectionWidget" name="psw_projection"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
  <customwidget>
   <class>DatastoreComboBox</class>
   <extends>QComboBox</extends>
   <header>vectiler.gui.cbx_datastore</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
